CMakeLists.txt

```diff
@@ -1,13 +1,9 @@
-# I have *literally* no idea what I'm doing.
-# Know any better? Feel free to correct my (probably) awful mistakes.
-# :)
-
 cmake_minimum_required(VERSION 3.1)
```

Comment:
Could you add compat code for CMake versions below 3.1? Right now the latest version of CMake for my system is 3.0.2 and I don't feel like pulling the package from elsewhere is a smart idea.